templates/web/common/site_layout.tt.html

```diff
@@ -367,9 +367,9 @@ <h3 class="title-5 text-medium">[% lang('footer_join_the_community') %]</h3>
 						<p>[% f_lang('f_join_us_on_slack', { url => "https://slack.openfoodfacts.org" } ) %]</p>
 						<p><a href="https://forum.openfoodfacts.org/">Forum</a></p>
 						<p id="footer_social_icons">[% f_lang('f_footer_follow_us_links', { links => '
-							<a href="https://twitter.com/' _ twitter_account _ '"><img src="/images/icons/dist/twitter.svg" class="footer_social_icon" alt="Twitter"></a>
-							<a href="' _ facebook_page_url _ '"><img src="/images/icons/dist/facebook.svg" class="footer_social_icon" alt="Facebook"></a>
-							<a href="https://www.instagram.com/open.food.facts/"><img src="/images/icons/dist/instagram.svg" class="footer_social_icon" alt="Instagram"></a>
+							<a href="https://x.com/OpenFoodFacts"><img src="/images/icons/dist/x.svg" class="footer_social_icon" alt="X"></a>
```

Comment:
We don't have `x.svg` in `icons/`. It would probably be best to add it in addition to https://github.com/openfoodfacts/openfoodfacts-server/blob/6456ca4775e69431eccf4d658acd07513b33cad0/icons/twitter.svg